client-sdk/go/modules/core/types.go

```diff
@@ -27,6 +27,7 @@ type GasCosts struct {
 // Parameters are the parameters for the consensus accounts module.
 type Parameters struct {
 	MaxBatchGas        uint64                                   `json:"max_batch_gas"`
+	MaxTxSize          uint32                                   `json:"max_tx_size,omitempty"`
```

Comment:
Maybe add a comment that this one has `omitempty` for backwards comparability, and can be removed later?